### ISSUE TITLE
Deleted files are not committed to the remote repo

### ISSUE THREAD

**Issue by AndyOGo** [semantic-release/git#155]
I have a library build baked by rollup with multiple entry points.
Rollup finds common dependencies and hashes them, so that after each release I end up with dirty files, being deleted but not commited:

```sh
On branch master
Your branch is ahead of 'origin/master' by 1 commit.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	deleted:    dist/confiig-93410e72.js
	deleted:    dist/definition-3fd983cb.js
```

New files are `dist/config-8a65f714.js` and `dist/definition-4e23ca7b.js`.

**Edit:**

`.releaserc`:
```json
{
  "plugins": [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    ["@semantic-release/changelog", {
      "changelogFile": "CHANGELOG.md"
    }],
    ["@semantic-release/npm", {
      "npmPublish": true,
    }],
    ["@semantic-release/git", {
      "assets": ["dist/**/*.{js,css}", "CHANGELOG.md", "package.json"],
      "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
    }]
  ],
  "branch": "master",
  "dryRun": false,
  "ci": false,
}

`package.json`: (only related)

```json
{
  "devdependencies": {
    "@semantic-release/changelog": "^3.0.4",
    "@semantic-release/git": "^7.0.16",
    "semantic-release": "^15.13.24"  
  }
}
```

**Comment by pvdlg** [comment]
Please provide the full logs and your semantic-release config.

**Comment by AndyOGo** [comment]
@pvdlg Thanks for your answer.
I edited my description above, unfortunately don't have the logs at hand 😞

**Comment by pvdlg** [comment]
This is an unfortunate side effect of #67 which was implemented due to some bugs in older version of `git` (see #62).

The problem is that we resolve the files matching the globs in `assets` first, then we check which one of those were modified and commit them. When we resolve the globs we miss the deleted files (as they are not there anymore) so we don't include them in the file we commit.

One solution would be to revert #67 but that would bring back the issue we had in the first place...

A workaround would be for you to configure `assets` with `dist` instead of `dist/**/*.{js,css}`. This way the deleted file should be included in the commit.

**Comment by AndyOGo** [comment]
@pvdlg  Thanks for your detailed answer 👍 

In #67 I could find [`--deleted`](https://git-scm.com/docs/git-ls-files#Documentation/git-ls-files.txt---deleted) being applied to `git ls-files` 🤔 

As I understand the docs of [`git ls-files`](https://git-scm.com/docs/git-ls-files#Documentation/git-ls-files.txt) without `-d/--deleted` they won't be included in the output.

**Comment by pvdlg** [comment]
The code uses `-m` which includes deleted files.

**Comment by AndyOGo** [comment]
Thanks. I'm curious though if **modified** includes deleted 🤔

**Comment by pvdlg** [comment]
I'll keep this issue open as it would have to be solved at some point.
Unfortunately I don't see any solution other than reverting #67. I guess we can do so when we increase the minimum version of `git` that we require.

**Comment by AndyOGo** [comment]
@pvdlg 
I have an idea, but it involves another overhead:
1) run `git ls-files -m -o` and filter by glob (to support that broken git version)
2) run `git ls-files --deleted` and add it to above (may check for duplicates)

**Comment by pvdlg** [comment]
The problem happens when you call `git ls-files` without specifying a list of file. The option passed (`-m`, `-d`, etc...)doesn't matters.

`git ls-files -d -m -o file1.js,file2.js` => works
`git ls-files -d -m -o` => create the problem described in #62

**Comment by AndyOGo** [comment]
@pvdlg Thanks for your quick reply and your patience.

Maybe another stupid question... How likely is it that `git ls-files --deleted` (without `-m -o`) creates described problem in #62 ?

**Comment by pvdlg** [comment]
`git ls-files --deleted` is as likely to create the problem from #62 as `git ls-files -m -o`.
That's what I meant by:
> The option passed (-m, -d, etc...)doesn't matters.

**Comment by AndyOGo** [comment]
Okay, then there needs to be a better way.

Currently I have this bug for 2 scenarious (without any `.gitmodules` files):

- `rm foo.txt`
- `mv foo.txt bar.txt`

Both will never be committed.

**Comment by pvdlg** [comment]
As explained before the solution is to revert #67. But doing so will make #62 to happens unless we require git >2.7.1.
We will do that at some point (probably next major release) but not now to avoid a regression.

**Comment by AndyOGo** [comment]
Thanks for taking your precious time to elaborate all these details.

Would it be a solution to make this behavior configurable?
I.e. enabling or disabling #67 by a flag?

**Comment by pvdlg** [comment]
No, we'll make the fix when we can require a non bugged version of `git`.

**Comment by AndyOGo** [comment]
Okay.
Can you give a rough plan, when this change would be feasible?

**Comment by AndyOGo** [comment]
Alright, just one more curiosity.

What if we wrap it within `try {} catch(error) { // #62 fix here only upon error }`?
This would lead to no breaking changes, still support old git versions and I would be happy too 😃 

Sorry if a bother you, but I really like to use this project and this is the only little thing stopping to do so...

**Comment by pvdlg** [comment]
Per https://github.com/semantic-release/git/issues/155#issuecomment-534422846 it seems you have a working workaround?

**Comment by AndyOGo** [comment]
Sorry for missunderstand, I edited my https://github.com/semantic-release/git/issues/155#issuecomment-534422846 to clarify what I answered (it was regarding `-m` option of `git ls-files`)

**Comment by pvdlg** [comment]
Have you tried the workaround I mentioned [there](https://github.com/semantic-release/git/issues/155#issuecomment-534255979) ?

**Comment by AndyOGo** [comment]
@pvdlg Thanks a lot, just using `dist` works for me:)

Below are the logs:

`npm run release -- --debug`:
```sh
> @axa-ch/aletheia-zoid@1.0.6 prerelease .../aletheia-zoid
> npm run build


> @axa-ch/aletheia-zoid@1.0.6 prebuild .../aletheia-zoid
> rimraf dist/ tmp/


> @axa-ch/aletheia-zoid@1.0.6 build .../aletheia-zoid
> rollup -c

node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead

src/index.js, src/component.js, src/frameworks.js, src/createReact.js → dist...
created dist in 1.2s

> @axa-ch/aletheia-zoid@1.0.6 release .../aletheia-zoid
> semantic-release "--debug"

[9:33:23 AM] [semantic-release] › ℹ  Running semantic-release version 15.13.24
  semantic-release:config load config from: .../aletheia-zoid/.releaserc +0ms
  semantic-release:config options values: { branch: 'master',
  semantic-release:config   repositoryUrl: 'git+https://github.com/axa-ch/aletheia-zoid.git',
  semantic-release:config   tagFormat: 'v${version}',
  semantic-release:config   plugins:
  semantic-release:config    [ [ '@semantic-release/commit-analyzer', [Object] ],
  semantic-release:config      '@semantic-release/release-notes-generator',
  semantic-release:config      [ '@semantic-release/changelog', [Object] ],
  semantic-release:config      [ '@semantic-release/npm', [Object] ],
  semantic-release:config      [ '@semantic-release/git', [Object] ] ],
  semantic-release:config   dryRun: false,
  semantic-release:config   ci: false,
  semantic-release:config   _: [],
  semantic-release:config   debug: true,
  semantic-release:config   '$0':
  semantic-release:config    '.../aletheia-zoid/node_modules/.bin/semantic-release',
  semantic-release:config   noCi: true } +2ms
  semantic-release:plugins options for @semantic-release/changelog/verifyConditions: { changelogFile: 'CHANGELOG.md' } +0ms
[9:33:23 AM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/changelog"
  semantic-release:plugins options for @semantic-release/npm/verifyConditions: { npmPublish: true } +1ms
[9:33:23 AM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/npm"
  semantic-release:plugins options for @semantic-release/git/verifyConditions: { assets: [ 'dist', 'CHANGELOG.md', 'package.json' ],
  semantic-release:plugins   message:
  semantic-release:plugins    'chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}' } +2ms
[9:33:23 AM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/git"
  semantic-release:plugins options for @semantic-release/commit-analyzer/analyzeCommits: { releaseRules: [ { type: 'refactor', release: 'patch' } ] } +1ms
[9:33:23 AM] [semantic-release] › ✔  Loaded plugin "analyzeCommits" from "@semantic-release/commit-analyzer"
  semantic-release:plugins options for @semantic-release/release-notes-generator/generateNotes: {} +0ms
[9:33:23 AM] [semantic-release] › ✔  Loaded plugin "generateNotes" from "@semantic-release/release-notes-generator"
  semantic-release:plugins options for @semantic-release/changelog/prepare: { changelogFile: 'CHANGELOG.md' } +1ms
[9:33:23 AM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/changelog"
  semantic-release:plugins options for @semantic-release/npm/prepare: { npmPublish: true } +0ms
[9:33:23 AM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/npm"
  semantic-release:plugins options for @semantic-release/git/prepare: { assets: [ 'dist', 'CHANGELOG.md', 'package.json' ],
  semantic-release:plugins   message:
  semantic-release:plugins    'chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}' } +0ms
[9:33:23 AM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/git"
  semantic-release:plugins options for @semantic-release/npm/publish: { npmPublish: true } +1ms
[9:33:23 AM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/npm"
[9:33:23 AM] [semantic-release] › ✔  Run automated release from branch master
[9:33:25 AM] [semantic-release] › ✔  Allowed to push to the Git repository
[9:33:25 AM] [semantic-release] › ℹ  Start step "verifyConditions" of plugin "@semantic-release/changelog"
[9:33:25 AM] [semantic-release] › ✔  Completed step "verifyConditions" of plugin "@semantic-release/changelog"
[9:33:25 AM] [semantic-release] › ℹ  Start step "verifyConditions" of plugin "@semantic-release/npm"
[9:33:25 AM] [semantic-release] [@semantic-release/npm] › ℹ  Verify authentication for registry https://registry.npmjs.org/
[9:33:26 AM] [semantic-release] › ✔  Completed step "verifyConditions" of plugin "@semantic-release/npm"
[9:33:26 AM] [semantic-release] › ℹ  Start step "verifyConditions" of plugin "@semantic-release/git"
[9:33:26 AM] [semantic-release] › ✔  Completed step "verifyConditions" of plugin "@semantic-release/git"
  semantic-release:get-last-release found tags: [ { gitTag: 'v1.0.6', version: '1.0.6' }, { gitTag: 'v1.0.5', version: '1.0.5' }, { gitTag: 'v1.0.4', version: '1.0.4' }, { gitTag: 'v1.0.3', version: '1.0.3' }, { gitTag: 'v1.0.2', version: '1.0.2' }, { gitTag: 'v1.0.1', version: '1.0.1' }, { gitTag: 'v1.0.0', version: '1.0.0' } ] +0ms
[9:33:27 AM] [semantic-release] › ℹ  Found git tag v1.0.6 associated with version 1.0.6
  semantic-release:get-commits Use gitHead: eea93dca225e62c2f47d96e9bae92334fd22940b +0ms
[9:33:27 AM] [semantic-release] › ℹ  Found 6 commits since last release
  semantic-release:get-commits Parsed commits: [ { commit: { long: 'c1ec108a9c8c3e2dbd18f6adccfbd4e08555db7a', short: 'c1ec108' }, tree: { long: '910ce0bd5e54c900ba193ea1dd48a9dc2ac0610b', short: '910ce0b' }, author: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:33:08.000Z }, committer: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:33:08.000Z }, subject: 'chore: fix missed comma in releaserc file', body: '', hash: 'c1ec108a9c8c3e2dbd18f6adccfbd4e08555db7a', message: 'chore: fix missed comma in releaserc file', gitTags: '(HEAD -> master, origin/master, origin/HEAD)', committerDate: 2019-09-26T07:33:08.000Z }, { commit: { long: 'fc40b791ef7db1eaa973a96042ddb3ea0d525013', short: 'fc40b79' }, tree: { long: 'eb66e29e28db2f3fa39c6ab6ebeea869db24a5ab', short: 'eb66e29' }, author: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:31:18.000Z }, committer: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:31:18.000Z }, subject: 'chore: also make a patch release for refactorings', body: '', hash: 'fc40b791ef7db1eaa973a96042ddb3ea0d525013', message: 'chore: also make a patch release for refactorings', gitTags: '', committerDate: 2019-09-26T07:31:18.000Z }, { commit: { long: '4adabc7514088455d30789f7540d48347df0dae1', short: '4adabc7' }, tree: { long: 'e95fb8d70b63a57fc57fc0cb48b5a9f85297a6e4', short: 'e95fb8d' }, author: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:25:09.000Z }, committer: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:25:09.000Z }, subject: 'chore: just set dist for assets', body: '', hash: '4adabc7514088455d30789f7540d48347df0dae1', message: 'chore: just set dist for assets', gitTags: '', committerDate: 2019-09-26T07:25:09.000Z }, { commit: { long: '3914ec26ce7e825d08593892d90d7b7a70912a5b', short: '3914ec2' }, tree: { long: '81ec09084012056b3ad305498971dfc655d9e5d1', short: '81ec090' }, author: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:23:55.000Z }, committer: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:23:55.000Z }, subject: 'refactor: prerender styles output', body: '', hash: '3914ec26ce7e825d08593892d90d7b7a70912a5b', message: 'refactor: prerender styles output', gitTags: '', committerDate: 2019-09-26T07:23:55.000Z }, { commit: { long: '94daafa508509746311e07cdebd40920a58ff867', short: '94daafa' }, tree: { long: '4864dc853097e0eff7efb1e27d884655a91db96e', short: '4864dc8' }, author: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:21:48.000Z }, committer: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:21:48.000Z }, subject: 'refactor: cache xprops access', body: '', hash: '94daafa508509746311e07cdebd40920a58ff867', message: 'refactor: cache xprops access', gitTags: '', committerDate: 2019-09-26T07:21:48.000Z }, { commit: { long: 'd69a3b18917ffa626d56e1b0d4408715ab4ac4b7', short: 'd69a3b1' }, tree: { long: '0672580c5cb5c6a38cb3c7588b74681d4f9779f8', short: '0672580' }, author: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:19:39.000Z }, committer: { name: 'Andreas Deuschlinger', email: '*@g*.*', date: 2019-09-26T07:19:39.000Z }, subject: 'refactor: moved callback assignment to top', body: '', hash: 'd69a3b18917ffa626d56e1b0d4408715ab4ac4b7', message: 'refactor: moved callback assignment to top', gitTags: '', committerDate: 2019-09-26T07:19:39.000Z } ] +21ms
[9:33:27 AM] [semantic-release] › ℹ  Start step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: chore: fix missed comma in releaserc file
  semantic-release:commit-analyzer Analyzing with custom rules +0ms
  semantic-release:commit-analyzer Analyzing with default rules +1ms
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The commit should not trigger a release
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: chore: also make a patch release for refactorings
  semantic-release:commit-analyzer Analyzing with custom rules +0ms
  semantic-release:commit-analyzer Analyzing with default rules +0ms
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The commit should not trigger a release
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: chore: just set dist for assets
  semantic-release:commit-analyzer Analyzing with custom rules +1ms
  semantic-release:commit-analyzer Analyzing with default rules +0ms
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The commit should not trigger a release
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: refactor: prerender styles output
  semantic-release:commit-analyzer Analyzing with custom rules +2ms
  semantic-release:commit-analyzer The rule { type: 'refactor', release: 'patch' } match commit with release type 'patch' +0ms
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The release type for the commit is patch
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: refactor: cache xprops access
  semantic-release:commit-analyzer Analyzing with custom rules +0ms
  semantic-release:commit-analyzer The rule { type: 'refactor', release: 'patch' } match commit with release type 'patch' +0ms
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The release type for the commit is patch
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: refactor: moved callback assignment to top
  semantic-release:commit-analyzer Analyzing with custom rules +0ms
  semantic-release:commit-analyzer The rule { type: 'refactor', release: 'patch' } match commit with release type 'patch' +1ms
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The release type for the commit is patch
[9:33:27 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analysis of 6 commits complete: patch release
[9:33:27 AM] [semantic-release] › ✔  Completed step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
[9:33:27 AM] [semantic-release] › ℹ  The next release version is 1.0.7
[9:33:27 AM] [semantic-release] › ℹ  Start step "generateNotes" of plugin "@semantic-release/release-notes-generator"
  semantic-release:release-notes-generator version: '1.0.7' +0ms
  semantic-release:release-notes-generator host: undefined +0ms
  semantic-release:release-notes-generator owner: 'axa-ch' +1ms
  semantic-release:release-notes-generator repository: 'aletheia-zoid' +0ms
  semantic-release:release-notes-generator previousTag: 'v1.0.6' +0ms
  semantic-release:release-notes-generator currentTag: 'v1.0.7' +0ms
  semantic-release:release-notes-generator host: 'https://github.com' +0ms
  semantic-release:release-notes-generator host: 'https://github.com' +0ms
  semantic-release:release-notes-generator linkReferences: undefined +0ms
  semantic-release:release-notes-generator issue: 'issues' +0ms
  semantic-release:release-notes-generator commit: 'commit' +0ms
[9:33:27 AM] [semantic-release] › ✔  Completed step "generateNotes" of plugin "@semantic-release/release-notes-generator"
[9:33:27 AM] [semantic-release] › ℹ  Start step "prepare" of plugin "@semantic-release/changelog"
[9:33:27 AM] [semantic-release] [@semantic-release/changelog] › ℹ  Update .../aletheia-zoid/CHANGELOG.md
[9:33:27 AM] [semantic-release] › ✔  Completed step "prepare" of plugin "@semantic-release/changelog"
[9:33:27 AM] [semantic-release] › ℹ  Start step "prepare" of plugin "@semantic-release/npm"
[9:33:27 AM] [semantic-release] [@semantic-release/npm] › ℹ  Write version 1.0.7 to package.json in .../aletheia-zoid
v1.0.7
[9:33:28 AM] [semantic-release] › ✔  Completed step "prepare" of plugin "@semantic-release/npm"
[9:33:28 AM] [semantic-release] › ℹ  Start step "prepare" of plugin "@semantic-release/git"
  semantic-release:git globed assets: [ 'dist/component.js', 'dist/config-c26377ba.js', 'dist/createReact.js', 'dist/definition-63a1d537.js', 'dist/frameworks.js', 'dist/index.js', 'dist', 'CHANGELOG.md', 'package.json' ] +0ms
[9:33:28 AM] [semantic-release] [@semantic-release/git] › ℹ  Found 10 file(s) to commit
  semantic-release:git add file to git index { stdout: '',
  stderr: '',
  code: 0,
  failed: false,
  killed: false,
  signal: null,
  cmd:
   'git add --force --ignore-errors dist/config-c26377ba.js dist/definition-63a1d537.js CHANGELOG.md dist/component.js dist/config-8a65f714.js dist/createReact.js dist/definition-4e23ca7b.js dist/frameworks.js dist/index.js package.json',
  timedOut: false } +0ms
  semantic-release:git commited files: [ 'dist/config-c26377ba.js', 'dist/definition-63a1d537.js', 'CHANGELOG.md', 'dist/component.js', 'dist/config-8a65f714.js', 'dist/createReact.js', 'dist/definition-4e23ca7b.js', 'dist/frameworks.js', 'dist/index.js', 'package.json' ] +39ms
[9:33:33 AM] [semantic-release] [@semantic-release/git] › ℹ  Prepared Git release: v1.0.7
[9:33:33 AM] [semantic-release] › ✔  Completed step "prepare" of plugin "@semantic-release/git"
[9:33:33 AM] [semantic-release] › ℹ  Start step "generateNotes" of plugin "@semantic-release/release-notes-generator"
  semantic-release:release-notes-generator version: '1.0.7' +6s
  semantic-release:release-notes-generator host: undefined +0ms
  semantic-release:release-notes-generator owner: 'axa-ch' +0ms
  semantic-release:release-notes-generator repository: 'aletheia-zoid' +0ms
  semantic-release:release-notes-generator previousTag: 'v1.0.6' +1ms
  semantic-release:release-notes-generator currentTag: 'v1.0.7' +0ms
  semantic-release:release-notes-generator host: 'https://github.com' +0ms
  semantic-release:release-notes-generator host: 'https://github.com' +0ms
  semantic-release:release-notes-generator linkReferences: undefined +0ms
  semantic-release:release-notes-generator issue: 'issues' +0ms
  semantic-release:release-notes-generator commit: 'commit' +0ms
[9:33:33 AM] [semantic-release] › ✔  Completed step "generateNotes" of plugin "@semantic-release/release-notes-generator"
[9:33:37 AM] [semantic-release] › ✔  Created tag v1.0.7
[9:33:37 AM] [semantic-release] › ℹ  Start step "publish" of plugin "@semantic-release/npm"
[9:33:37 AM] [semantic-release] [@semantic-release/npm] › ℹ  Publishing version 1.0.7 to npm registry
npm notice
npm notice 📦  @axa-ch/aletheia-zoid@1.0.7
npm notice === Tarball Contents ===
npm notice 600B    .babelrc
npm notice 15B     .browserslistrc
npm notice 414B    .editorconfig
npm notice 22B     .eslintignore
npm notice 105B    .huskyrc
npm notice 46B     .lintstagedrc
npm notice 22B     .prettierignore
npm notice 115B    .prettierrc
npm notice 604B    .releaserc
npm notice 39B     .sassrc
npm notice 70B     commitlint.config.js
npm notice 2.5kB   dist/component.js
npm notice 4.4kB   dist/config-c26377ba.js
npm notice 264B    dist/createReact.js
npm notice 382.0kB dist/definition-63a1d537.js
npm notice 189.6kB dist/frameworks.js
npm notice 143B    dist/index.js
npm notice 2.5kB   rollup.config.js
npm notice 1.8kB   .eslintrc.json
npm notice 2.5kB   package.json
npm notice 3.9kB   CHANGELOG.md
npm notice 5.0kB   README.md
npm notice === Tarball Details ===
npm notice name:          @axa-ch/aletheia-zoid
npm notice version:       1.0.7
npm notice package size:  110.6 kB
npm notice unpacked size: 596.5 kB
npm notice shasum:        aad473bfe0ffd4a795af4c67f6cff4ec8d8d4453
npm notice integrity:     sha512-B2KS+pESjxBCV[...]UAt+UQNcNySpw==
npm notice total files:   22
npm notice
+ @axa-ch/aletheia-zoid@1.0.7
[9:33:43 AM] [semantic-release] [@semantic-release/npm] › ℹ  Published @axa-ch/aletheia-zoid@1.0.7 on https://registry.npmjs.org/
[9:33:43 AM] [semantic-release] › ✔  Completed step "publish" of plugin "@semantic-release/npm"
[9:33:43 AM] [semantic-release] › ✔  Published release 1.0.7
```

And `git status`:
```sh
On branch master
Your branch is ahead of 'origin/master' by 1 commit.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
```

**Comment by pvdlg** [comment]
Closing as it was fixed by #168 on `beta` branch. Unfortunately this can't be fixed without a breaking change as the fix would require Git `2.7.1` (we currently require `2.0.0`)
